ynput/ayon-core
# Chore: Move fusion client code

## Changelog Description
Moved fusion client code next to server codebase.

## Testing notes:
1. Create package of fusion and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Fusion integration should work as before.
